pages/common/tsv-filter.md

```diff
@@ -0,0 +1,36 @@
+# tsv-filter
+
+> Filter lines of a TSV file by running tests against individual fields.
+> More information: <https://github.com/eBay/tsv-utils#tsv-filter>.
+
+- Print the lines where a specific column is numerically equal to a given number:
+
+`tsv-filter -H --eq {{field_name}}:{{number}} {{path/to/tsv_file}}`
+
+- Print the lines where a specific column is [eq]ual/[n]on [e]qual/[l]ess [t]han/[l]ess than or [e]qual/[g]reater [t]han/[g]reater than or [e]qual to a given number:
```

Comment:
This is an eyesore IMO. And I think its quite unreadable.

Anyone got any better solutions?

Reply:
one could not mention all the options - on the other hand it's exactly these sorts of things that I would personally use `tldr` for.

I'm a newcomer here so happy for the team to decide on whatever you prefer. Something is better than nothing in any case in my eyes.

Reply:
Lets stick with the current one, if it gives all of the necessary info across to the user then I guess it is fine, even if it is a bit of a hard time to read it.

Reply:
I think I agree with @CleanMachine1 here - this is the sort of thing where having all the info saves one from having to go to the man page.

Reply:
Sometimes there is literally nothing you can do. Take the `whiptail` page for example. The examples are horrible - but there is simply no other way to do it.

That said, being wary of cramming too much into a single example isn't necessarily a good thing, as it can detract from the clarity of the page. I can see arguments for both sides here. Which one would be most useful to users of this command? Does anyone know some  people we could consult (either IRL or online)?

Reply:
I'm a real life user of this tool - I often have to look at help for these exact commands. There isn't a most useful one of equality etc - they are all needed and relevant.

Reply:
> `tldr whiptail` doesn't seem to exist :o


It does for me, when did you last update your cache?
https://github.com/tldr-pages/tldr/blob/main/pages/linux/whiptail.md